GamesProject/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>GamesProject</title>
+    <link rel="stylesheet" href="../GamesProject/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Inter:wght@500;700&family=Roboto:wght@400;700&family=Young+Serif&display=swap" 
+rel="stylesheet">
+</head>
+
+<body>
+
+    <main>
+        <section class="content">
+            <div class="about">
+            <h1 class="text-title">About me</h1>
+                <p>
+                    Hi! My name is Andrew and I'm a Junior Frontend Developer. 
+                    I am already familiar with main Web Technologies like HTML, CSS, 
+                    JavaScript and Git version control system.
+                </p> 
+                <p>
+                    This page was developed during the course
+                </p>
+                <p>
+                    <a href="https://www.mastersacademy.education/frontend-for-beginners-it">'Frontend for beginners' from Masters Academy in 2023</a>
```

Comment:
The link looks inline on the design. So it is unnecessary to wrap it into the `p`
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/49fad329-cae7-4450-9150-066b19b473d7)